### PR TITLE
update:  badge

### PR DESCRIPTION
**Pull request by rahulcs754**
image

Please check and review

### CODE REVIEW

File: css/components/badge.css [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+*/
+.badge {
+    position: relative;
+} 
+
+.badge-img {
+    border-radius: 50%;
+    width: 2.5rem;
+    height: 2.5rem;
+    margin: 0 10px;
```
Comment (SimranjitKaurMaan) [inline css/components/badge.css]: You can use `rem` instead of `px`.

File: css/components/badge.css [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+    position: absolute;
+    font-size: 0.9rem;
+    color: var(--mint-white);
+    font-weight: 900;
+    height: 1.125rem;
+    width: 1.125rem;
+    border: none;
+    background-color: var(--primary-color);
+    border-radius: 50%;
+    padding: 14px;
```
Comment (SimranjitKaurMaan) [inline css/components/badge.css]: Same as above.

File: css/components/badge.css (hunk 1 above)
Comment (rahulcs754) [reply css/components/badge.css]: done

File: css/components/badge.css (hunk 2 above)
Comment (rahulcs754) [reply css/components/badge.css]: done

File: css/components/badge.css [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+
+.badge-img {
+    border-radius: 50%;
+    width: 2.5rem;
+    height: 2.5rem;
+    margin: 0 10px;
+}
+
+.badge-icon {
+    margin: 0 10px;
```
Comment (shubhamkumar648) [inline css/components/badge.css]: here also you used rem

**Review by shubhamkumar648 — APPROVED** [review]
minor changes need otherwise good to go

**Review by aatishpandey — COMMENTED** [review]
Looks good to go

File: css/components/badge.css (hunk 3 above)
Comment (rahulcs754) [reply css/components/badge.css]: done

**Review by SimranjitKaurMaan — APPROVED** [review]

**Pull request merged**